Tasks/AWSShellScript/TaskOperations.ts

```diff
@@ -49,19 +58,49 @@ export class TaskOperations {
             }
 
             const execOptions = {
+                cwd: workingDirectory,
                 env: process.env,
+                errStream: process.stdout,
+                outStream: process.stdout,
                 failOnStdErr: this.taskParameters.failOnStandardError
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            return await bash.exec(execOptions as tr.IExecOptions)
+            return await bash.exec((execOptions as unknown) as tr.IExecOptions)
```

Comment:
If we declare `execOptions` as `tr.IExecOptions`, would it eliminate the need for the "as unknown as x" cast?

Reply:
No, because then it is missing fields from the interface. We could define them as undefined to keep the same behavior, but then strict mode will be mad.